isis/src/base/objs/Camera/Camera.cpp

```diff
@@ -1540,9 +1540,9 @@ namespace Isis {
       // order of points in vector is top, bottom, left, right
       QList< QPair< double, double > > surroundingPoints;
       surroundingPoints.append(qMakePair(samp, line - 0.5));
-      surroundingPoints.append(qMakePair(samp, line + 0.5 - DBL_MIN));
+			surroundingPoints.append(qMakePair(samp, std::nexttoward(line + 0.5, line)));
       surroundingPoints.append(qMakePair(samp - 0.5, line));
-      surroundingPoints.append(qMakePair(samp + 0.5 - DBL_MIN, line));
+      surroundingPoints.append(qMakePair(std::nexttoward(samp + 0.5, samp), line));
```

Comment:
These two changes make sense, and will fix the problem for phocube. If you want a slightly more accurate answer, you should adjust line 1542 and 1544 in a similar way as you did for 1543 and 1545. This will keep the requested pixel (line/sample) centered between the four surrounding pixels.

Although this does fix this function when phocube uses it with integer sample/line values, it does not fix it when other applications use non-integer sample/line values. We have created a new issue #4256 to address what we believe to be a fix for the other cases.

As far as testing this change goes: We do not believe it is reasonable to test this change under the current unittest environment. This would be much more reasonable once the camera test has been converted to gtest. Given that, this PR should be mergeable after addressing the current comments.

Reply:
Working to implement changes. Thanks.